plugin.video.mlbbasesloaded/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.mlbbasesloaded" name="MLB BasesLoaded" version="0.2" provider-name="Jake Kramer">
+<addon id="plugin.video.mlbbasesloaded" name="MLB BasesLoaded" version="0.3" provider-name="Jake Kramer">
```

Comment:
Sorry, but it seems like we missed this the last times, version number should be 0.3.0 so three parts.
http://kodi.wiki/view/Addon.xml#version_attribute